e2e/tests/121-Portal/EditInfoPersonAffected/ViewInformationIconInTable.spec.ts

```diff
@@ -0,0 +1,38 @@
+import HomePage from '@121-e2e/pages/Home/HomePage';
+import LoginPage from '@121-e2e/pages/Login/LoginPage';
+import TableModule from '@121-e2e/pages/Table/TableModule';
+import NLRCProgram from '@121-service/seed-data/program/program-nlrc-ocw.json';
+import { SeedScript } from '@121-service/src/scripts/seed-script.enum';
+import { seedPaidRegistrations } from '@121-service/test/helpers/registration.helper';
+import { resetDB } from '@121-service/test/helpers/utility.helper';
+import { registrationsOCW } from '@121-service/test/registrations/pagination/pagination-data';
+import { test } from '@playwright/test';
+
+test.beforeEach(async ({ page }) => {
+  await resetDB(SeedScript.nlrcMultiple);
+  const programIdOCW = 3;
+  const OcwProgramId = programIdOCW;
+
+  await seedPaidRegistrations(registrationsOCW, OcwProgramId);
+
+  // Login
+  const loginPage = new LoginPage(page);
+  await page.goto('/login');
+  await loginPage.login(
+    process.env.USERCONFIG_121_SERVICE_EMAIL_ADMIN,
+    process.env.USERCONFIG_121_SERVICE_PASSWORD_ADMIN,
+  );
+});
+
+test('[28035] View information icon in table', async ({ page }) => {
+  const table = new TableModule(page);
+  const homePage = new HomePage(page);
+
+  await test.step('Navigate to PA table', async () => {
+    await homePage.navigateToProgramme(NLRCProgram.titlePortal.en);
+  });
+
+  await test.step('Validate information icons are available in the table', async () => {
+    await table.validateInformationButtonsPresent({ expectedCount: 4 });
```

Comment:
Would this number `4` be the `registrationsOCW.length`?

It could make sense to 'fix' the number to this hard-coded value, but sometimes I also like that the code is 'resilient to change'; so that you don't _have to_ change this value only because 1 other test-registration is added to the set (because of totally unrelated reasons)...

Reply:
Kind of made sense to me to put it like that just in case seed function will change but if you believe it is better to keep it fixed I can change it.

Reply:
My point was sort-of the other way around :)

I prefer to use "the _least_ amount of really specific numbers/strings in code"(so a number `4`, or some `string`) and **always** use "the _most_ descriptive/specific way to define something" (so using a variable/property/piece of working code; like this `.length`-property, or something like: `60 * 60 * 24 * 7` to describe 'the number of seconds in a week', instead of `604800`)